doc/api/util.md

```diff
@@ -107,30 +107,50 @@ environment variable set, then it will not print anything.
 Multiple comma-separated `section` names may be specified in the `NODE_DEBUG`
 environment variable. For example: `NODE_DEBUG=fs,net,tls`.
 
-## util.deprecate(function, string)
+## util.deprecate(fn, msg[, code])
```

Comment:
nit: `s/fn/function`, `s/msg/message`

Reply:
I changed it to `fn` to match the code. `function` is  a keyword so we can't use that...

Reply:
Not looking for a debate here, but do want to make sure we have broad agreement and perhaps something that needs an entry in the style guide or something:

If I'm not mistaken, we try to use the same argument name in the docs as we use for the variables in the source code.

I believe there are two reasons, but correction welcome. First, it makes the reading of source code along with docs easier. Second, it allows the mapping of documentation variable names to variable names in stack traces possible.

If I'm right about that, then we can not use `function` as it is a keyword and not a valid variable name. We use `fn`  and `msg` elsewhere in the docs. I'm inclined to leave them. They are pretty self-explanatory, and get an explanation immediately below anyway.